app/src/main/res/layout/fragment_register.xml

```diff
@@ -0,0 +1,159 @@
+
+
+        />
+
+    <EditText
+        android:id="@+id/name"
+        android:layout_width="@dimen/name_width_250dp"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:drawableLeft="@drawable/ic_user_name"
+        android:hint="@string/username"
+        android:textColorHint="@color/color_FFF7F1F4"
+        android:inputType="text"
+        android:padding="@dimen/name_padding"
+        android:textSize="13sp"
+        android:layout_marginLeft="@dimen/name_marginleft"
+        android:backgroundTint="@color/color_FFF8F7F7"
+
+
+        />
+
+    <EditText
+        android:id="@+id/email"
+        android:layout_width="@dimen/email_width_250dp"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:drawableLeft="@drawable/ic_email"
+        android:hint="@string/e_mail"
+        android:textColorHint="@color/color_FFF7F1F4"
+        android:inputType="textEmailAddress"
+        android:padding="@dimen/email_padding"
+        android:textSize="13sp"
+        android:layout_marginLeft="@dimen/email_marginleft"
+        android:backgroundTint="@color/color_FFF8F7F7"
+
+
+
+
+        />
+
+    <EditText
+        android:id="@+id/password"
+        android:layout_width="@dimen/pass_width"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:drawableLeft="@drawable/ic_pass"
+        android:hint="@string/password"
+        android:textColorHint="@color/color_FFF7F1F4"
+        android:inputType="numberPassword"
+        android:padding="@dimen/pass_padding"
+        android:textSize="13sp"
+        android:layout_marginLeft="@dimen/pass_marginleft"
+        android:backgroundTint="@color/color_FFF8F7F7"
+
+
+
+        />
+
+    <EditText
+        android:id="@+id/confirm_password"
+        android:layout_width="@dimen/confirm_pass_width"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:drawableLeft="@drawable/ic_pass"
+        android:hint="@string/confirm_password"
+        android:textColorHint="@color/color_FDFBFB"
+        android:inputType="numberPassword"
+        android:padding="@dimen/confirm_pass_padding"
+        android:textSize="13sp"
+        android:layout_marginLeft="@dimen/confirm_pass_marginleft"
+        android:backgroundTint="@color/color_FFF8F7F7"
+
+
+        />
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/height_19"
+        android:layout_marginTop="@dimen/marginTop_25"
+        android:orientation="horizontal"
+        android:layout_marginLeft="@dimen/marginleft_75"
+        >
+
+        <CheckBox
+            android:id="@+id/cb_reg"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:buttonTint="@color/color_fff"
+
+
+            />
+
+        <TextView
```

Comment:
string has typo and checkbox design not matching design

Reply:
this is optional for now but in real projects we have to match the design with 100% accuracy